hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -126,14 +122,6 @@ func (mh *Memcached) Run() {
 	_, err = mh.ctx.Run(cmd)
 	pkg.CheckError("Running go mod tidy", err)
 
-	log.Infof("creating the bundle")
-	err = mh.ctx.GenerateBundle()
-	pkg.CheckError("creating the bundle", err)
-
-	log.Infof("striping bundle annotations")
-	err = mh.ctx.StripBundleAnnotations()
-	pkg.CheckError("striping bundle annotations", err)
```

Comment:
@NikhilSharmaWe this ones we need to keep 
They are specific for the SDK
We are creating here the bundle